Python/crossinterp.c

```diff
@@ -1692,6 +1692,47 @@ _PyXI_HasCapturedException(_PyXI_session *session)
     return session->error != NULL;
 }
 
+static int
+_PyXI_ThreadStateRecovery(void *ptr)
```

Comment:
Hmmm, I'm neutral on this, is using **`void *`** really a good idea? Maybe it has hidden memory errors.

Reply:
It's a callback, we have to use `void *`.